backend/src/test/java/planit/people/preparation/TestUtils.java

```diff
@@ -0,0 +1,49 @@
+package planit.people.preparation;
+
+import planit.people.preparation.Entities.*;
+
+import java.sql.Time;
+
+public class TestUtils {
+
+    public static Entity_Guest createGuest(String email, Boolean obligatory) {
+        return createGuest(email, obligatory, null);
+    }
+
+    public static Entity_Guest createGuest(String email, Boolean obligatory, Long id) {
+        Entity_Guest guest = new Entity_Guest(email, obligatory);
+        if (id != null) {
+            guest.setId_event_guest(id);
+        } else {
+            guest.setEntity_EventPreset(null);
+            guest.setId_event_guest(null);
+        }
+        return guest;
+    }
+
+    public static Entity_PresetAvailability createPresetAvailability(Entity_PresetAvailability.WeekDays day, Time start_available_time, Time end_available_time, Boolean day_off, Long id) {
```

Comment:
Why not `camelCase` in `start_available_time` etc.?

Reply:
these fields are exchanged between FE and BE and we decided to go with snake_case for this type of communication